lib/common/strings.c

```diff
@@ -97,7 +97,7 @@ pcmk__scan_ll(const char *text, long long *result, long long default_value)
 
     if (text != NULL) {
         rc = scan_ll(text, &local_result, default_value, NULL);
-        if (rc != pcmk_rc_ok) {
+        if ((rc != pcmk_rc_ok) && (rc != ERANGE)) {
```

Comment:
I think this block can be dropped, as `scan_ll()` already sets `local_result = default_value` in the same cases (double-check that).

Reply:
nice